### PR TITLE
Restructure, Remove and Renew Documentation.

### PR DESCRIPTION
**Pull request by gtrufitt**
## What does this change?

We have lots of documentation for Frontend - some in a /docs folder, some in Wiki, some in code.

This is an initial pass at restructuring the documentation so that it is in the codebase. We have moved wiki docs into the docs/ folder, removed un-needed docs and renewed some of them.
## What is the value of this and can you measure success?

Developers are able to find helpful, up-to-date documentation in one place.
## Request for comment

@guardian/dotcom-platform @guardian/guardian-frontend

### CODE REVIEW

**Comment by johnduffell** [comment]
😢 too many files to display!

**Comment by johnduffell** [comment]
https://github.com/guardian/frontend/blob/285214edcf390a7cfc56c112855c6943be43f595/docs/01-start-here/README.md is empty, should it be the main document in there?

File: docs/01-start-here/Quick visual glossary of the Guardian dot com.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
+# Quick visual glossary of the Guardian dot com
```
Comment (johnduffell) [inline docs/01-start-here/Quick visual glossary of the Guardian dot com.md]: there is also some useful.duplicate info in great detail here: https://sites.google.com/a/guardian.co.uk/next-gen-fronts-design/home

**Comment by TBonnin** [comment]
@johnduffell Yes the README will contain a table of content of the different sections and docs file. 
That would be step 2 of the documentation overhaul. Stay tuned for another PR ;)

**Comment by johnduffell** [comment]
great idea, looks really good, and moving stuff to keep it up to date is great!  my only 😞 is that we've lost the history although for docs is less problematic.  apparently git should notice renames and keep the history, but clearly it didn't
👍

**Comment by gtrufitt** [comment]
@johnduffell It has kept history for files that were in the repo:

image

It's only showing the new files as ones that have been moved across from the wiki. :)

**Pull request merged**